test/built-ins/TypedArrays/from/arylk-to-length-error.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+id: sec-%typedarray%.from
+description: Returns error produced by interpreting length property as a length
+info: >
+  22.2.2.1 %TypedArray%.from ( source [ , mapfn [ , thisArg ] ] )
+
+  ...
+  7. Let len be ? ToLength(? Get(arrayLike, "length")).
+  ...
+features: [Symbol.toPrimitive]
+includes: [testTypedArray.js]
+---*/
+
+var arrayLike = { length: {} };
+
+arrayLike.length[Symbol.toPrimitive] = function() {
```

Comment:
You can simplify this test a bit by defining the `length` property as `{ valueOf: function() { throw new Test262Error(); } }`